### DISCUSSION TITLE
Make generation of valhalla.json optional

### DISCUSSION THREAD

**Discussion by brad-4k** [gis-ops/docker-valhalla#85] [Ideas]
Hello,

We are running multiple instances, all sharing a network mounted volume that is mapped to /custom_files which contains the valhalla.json file.  The problem we are having is that when all the instances start, they are all trying to regenerate the valhalla.json file and at times, some fail because they are all trying to write to the file.

What is the reason for generating the json file?

It should be a simple change to /valhalla/scripts/configure_valhalla.sh to check if the file exists and not generate a new one if it does.

What are your thoughts?

**Comment by nilsnolde** [comment]
Our expertise is proprietary but can be purchased under enquiry@gis-ops.com.

**Reply by brad-4k** [reply]
Thank you Nils for the response.

I should have been a little more clear in my question.  The main point was to find out if this would break other peoples use cases.

I'm happy to make the change and commit but if this wouldn't be considered should I just fork the repo?

**Reply by nilsnolde** [reply]
Ah right, if that was the idea, then jep, I'd recommend forking.
